src/panels/config/integrations/integration-panels/thread/thread-config-panel.ts

```diff
@@ -160,19 +161,24 @@ export class ThreadConfigPanel extends SubscribeMixin(LitElement) {
   }
 
   private _renderNetwork(network: ThreadNetwork) {
+    const otbrForNetwork =
+      this._otbrInfo &&
+      network.dataset &&
+      ((network.dataset.preferred_extended_address &&
+        this._otbrInfo[network.dataset.preferred_extended_address]) ||
+        Object.values(this._otbrInfo).find(
+          (otbr) => otbr.extended_pan_id === network.dataset?.extended_pan_id
```

Comment:
```suggestion
          (otbr) => otbr.extended_pan_id === network.dataset!.extended_pan_id
```

Reply:
I am kinda surprised that the linter wouldn't let me use `network.dataset.extended_pan_id` here. I clearly check for `network.dataset` above :man_shrugging: